presto-main/src/test/java/com/facebook/presto/sql/planner/TestLogicalPlanner.java

```diff
@@ -482,12 +482,12 @@ public void testLeftConvertedToInnerInequalityJoinNoEquiJoinConjuncts()
     {
         assertPlan("SELECT 1 FROM orders o LEFT JOIN lineitem l ON o.orderkey < l.orderkey WHERE l.orderkey IS NOT NULL",
                 anyTree(
-                        filter("O_ORDERKEY < L_ORDERKEY",
-                                join(INNER, ImmutableList.of(), Optional.empty(),
-                                        tableScan("orders", ImmutableMap.of("O_ORDERKEY", "orderkey")),
-                                        any(
-                                                filter("NOT (L_ORDERKEY IS NULL)",
-                                                        tableScan("lineitem", ImmutableMap.of("L_ORDERKEY", "orderkey"))))))));
+                        project(filter("O_ORDERKEY < L_ORDERKEY",
```

Comment:
1. formatting nit: can you put the project on its own line (here and in the new predicate pushdown tests) so its easier to see the shape of the tree
2. is the project in the plan new or is the change just showing more explicitly that it gets added for the post-join filter case?

Reply:
1. Addressed the nit (mostly)
2. Its just to be more explicit about adding the extra project

Will wait for tests to pass and merge.